### ISSUE TITLE
[Suggestion]

### ISSUE THREAD

**Issue by JohnVKS** [ejektaflex/Bountiful#96]
## Add bount drop from fishing

Add some rare bounties from fishing, I think it can be nice if  the player is not really into finding villages

**Comment by ejektaflex** [comment]
Like, you can get a bounty as treasure from fishing? That's an interesting idea. I might look into it, but that would also require modifying loot pools so we'll see.

**Comment by JohnVKS** [comment]
Yeah, maybe make them more focusing on fishing and/or aquatic creatures

**Comment by ejektaflex** [comment]
I mean, a fisherman decree already exists so you can also do this with regular bounty boards too :^) but being able to fish up a fisherman bounty might be neat.

**Comment by ejektaflex** [comment]
I'm putting most of my new development work into the Fabric version of the mod right now, so I'll be transferring this issue to there.
